andino_firmware/src/hw.h

```diff
@@ -39,9 +39,9 @@ struct Hw {
   static constexpr int kLeftEncoderChannelBGpioPin{3};
 
   /// @brief Right encoder channel A pin. Connected to PC4 (digital pin 18, analog pin A4).
-  static constexpr int kRightEncoderChannelAGpioPin{18};
+  static constexpr int kRightEncoderChannelAGpioPin{14};
```

Comment:
Please open a ticket and add a TODO tag to the [andino_hardware README file](https://github.com/Ekumen-OS/andino/blob/humble/andino_hardware/README.md) so we can keep track of modifying the hardware diagram.